Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -652,30 +656,52 @@ void FieldProbe::ComputeDiags (int step)
 
 void FieldProbe::WriteToFile (int step) const
 {
-    if (ProbeInDomain() && amrex::ParallelDescriptor::IOProcessor())
+    if (!(ProbeInDomain() && amrex::ParallelDescriptor::IOProcessor())) return;
+
+    // loop over num valid particles to find the proper starting ID
```

Comment:
You mean the "lowest" id of a probe particle?

Reply:
That is correct yes.

Reply:
I would make the comment more explicit then. How about:
```suggestion
    // loop over num valid particles to find the lowest particle ID for later sorting
```